docs/Developers/Newcomers_Guide.rst

```diff
@@ -82,29 +82,22 @@ What documentation was missing? What was hard to understand?
 Once you complete this, please take the time and
 `fill this form <https://coala.io/usability>`_ so we can improve this!
 
-Step 1. Meet the Community!
----------------------------
+Step 1. Meet the Community and Get an Invitation to the Organization
+--------------------------------------------------------------------
 
 To get started, the first step is to meet the community. We use gitter to
-communicate, and there the helpful community will guide you.
-Join us at `coala gitter <https://coala.io/chat>`_.
-The newcomers should ping us "Hello World" to let us know they are here
-because we care!
-
-**Congratulations!** You are now part of our community.
-
-Step 2. Grab an Invitation to the Organization
-----------------------------------------------
+communicate, and there the helpful community will guide you. Join us at
+`coala gitter <https://coala.io/chat>`_. The newcomers should ping us "Hello
+World" to let us know they are here because we care!
 
-Let us know on gitter that you are interested in contributing and ask for an
-invitation to our org. This is your first step towards contributing.
-A maintainer will command ``cobot`` (our gitter bot) to invite
-you and be part of the Newcomer team.
-The invitation will be sent by mail and you will have to accept
-it to join. If you don't find the invitation, accept it `here <https://github.com/coala>`__.
+When you say "Hello World" in chat ``cobot`` (our gitter bot) will invite you
```

Comment:
chat -> the chatroom
The line should read:
When you say "Hello World" in the chatroom, ``cobot`` (our gitter bot) will invite you